### PR TITLE
OCI Source Type Added

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-20351

### CODE REVIEW

**Comment by fhlavac** [comment]
:eyes:

**Comment by fhlavac** [comment]
@radekkaluzik UI looks good just thinking about the review step 
image
In the mockups, there is no the blue User Access alert present. Is it needed?

Will go through the code now...

File: src/test/components/formComponents/sourceWizardSummary.test.js [hunk 1]
```diff
@@ -179,6 +179,41 @@ describe('SourceWizardSummary component', () => {
...
+      const { container } = render(<SourceWizardSummary {...initialProps} formOptions={formOptions} />);
+
+      const data = getListData(container);
+
+      expect(data).toEqual([
+        ['Name', 'source_name'],
+        ['Source type', 'Oracle Cloud Infrastructure'],
+        ['Application', 'Cost Management'],
+        ['Bucket', 'cost-management'],
+        ['Bucket namespace', 'yyhddfgeeu'],
```
Comment (Hyperkid123) [inline src/test/components/formComponents/sourceWizardSummary.test.js:212]: Can we use a slightly more descriptive value for the text?

**Review by Hyperkid123 — COMMENTED** [review]
Only a small nitpick. UI works fine.

File: src/test/components/formComponents/sourceWizardSummary.test.js (hunk 1 above)
Comment (radekkaluzik) [reply src/test/components/formComponents/sourceWizardSummary.test.js:212]: I used the same they are using in the mocks.

File: src/components/addSourceWizard/hardcodedComponents/oci/costManagement.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        })}
+      </Text>
+      <TextList className="pf-u-ml-0 pf-u-pl-md" component={TextListVariants.ol}>
+        <TextListItem className="pf-u-mb-lg">
+          {intl.formatMessage({
+            id: 'cost.oci.policyCompartment.subtitle1',
+            defaultMessage: 'Create cost and usage reports policy using the following command',
+          })}
+          <ClipboardCopy className="pf-u-mt-sm" variant={ClipboardCopyVariant.expansion}>
+            {`oci iam policy create --compartment-id ${values?.application?.extra?.compartment_id} --description "test" --name "test" --statements '["define tenancy usage-report as ocid1.tenancy.oc1..aaaaaaaaned4fkpkisbwjlr56u7cj63lf3wffbilvqknstgtvzub7vhqkggq","endorse group Administrators to read objects in tenancy usage-report"]'`}
```
Comment (fhlavac) [inline src/components/addSourceWizard/hardcodedComponents/oci/costManagement.js:94]: Also, this `aaaaaaaaned4fkpkisbwjlr56u7cj63lf3wffbilvqknstgtvzub7vhqkggq` looks a bit weird, I know it is in the mockups but we should maybe try to find out how it should be cleaned a bit and more clear for the user

File: src/components/addSourceWizard/hardcodedSchemas.js [hunk 3]
```diff
@@ -1239,6 +1240,151 @@ const hardcodedSchemas = {
...
+                    {
+                      name: 'application.extra.bucket_namespace',
+                      label: 'New data bucket namespace',
+                      validate: [
+                        {
+                          type: 'required',
+                        },
+                      ],
+                      component: 'text-field',
+                      placeholder: 'yyhddfgeeu',
```
Comment (fhlavac) [inline src/components/addSourceWizard/hardcodedSchemas.js:1349]: also this^^

**Comment by fhlavac** [comment]
@radekkaluzik looks really good! :tada: 

could you also enhance test coverage for
`EnhancedRadio.js`
and
`hardcodedComponents/oci/costManagement.js` ?

File: src/test/components/formComponents/sourceWizardSummary.test.js (hunk 1 above)
Comment (radekkaluzik) [reply src/test/components/formComponents/sourceWizardSummary.test.js]: Fixed

**Comment by fhlavac** [comment]
@radekkaluzik LGTM! :tada: 
Thanks for the tests!

**Review by fhlavac — APPROVED** [review]

**Pull request merged**